ios/brave-ios/Sources/AIChat/Constants/AIChatConstants.swift

```diff
@@ -6,7 +6,9 @@
 import Foundation
 
 public struct AIChatConstants {
-  static let braveLeoWikiURL = URL(string: "https://github.com/brave/brave-browser/wiki/Brave-Leo")!
+  static let braveLeoHelpCenterURL = URL(
```

Comment:
Dupe of `braveLeoSearchLearnMore` below?

Reply:
Yeah. I wasn't sure if to use the exact same URL, in case they updated it in the future?